HW1/P4/P4_bfs.py

```diff
@@ -0,0 +1,83 @@
+
+import pyspark
+from pyspark import SparkContext
+import time
+import sys
+
+#following function does BFS
+def BFS(sc, graph, source, maxITER=10):
+	#initialize. The nodes have a key (x) which is the character name
```

Comment:
Cool, I actually implemented the BFS in the same way with the unvisited flag. You are the only other person I have seen do it this way.